lib/cloud_foundry.rb

```diff
@@ -11,6 +11,15 @@ def self.vcap_data
     end
   end
 
+  def self.app_url
+    if self.is_environment?
+      # (arbitrarily) use the shortest assigned route, if there are multiple
+      self.vcap_data['application_uris'].min_by(&:length)
+    else
+      nil
```

Comment:
> For methods where the return value is important, I always do,

@afeld what would be an example of a method where the return value is not important? 

@pkarman interesting to hear that this can be confusing to people. I definitely prefer explicitness to concision but have never seen people explicitly return `nil` as part of that effort in Ruby!